src/Lykke.ExternalExchangesApi/Exchanges/Bitfinex/RestClient/BitFinexServiceClientCredentials.cs

```diff
@@ -6,20 +6,20 @@
 using System.Threading.Tasks;
 using Microsoft.Rest;
 
-namespace TradingBot.Exchanges.Concrete.Bitfinex
+namespace Lykke.ExternalExchangesApi.Exchanges.Bitfinex.RestClient
 {
-    internal sealed class BitfinexServiceClientCredentials : ServiceClientCredentials
+    public sealed class BitfinexServiceClientCredentials : ServiceClientCredentials
     {
         private const string ApiBfxKey = "X-BFX-APIKEY";
         private const string ApiBfxPayload = "X-BFX-PAYLOAD";
         private const string ApiBfxSig = "X-BFX-SIGNATURE";
 
-        private readonly string _apiKey;
+        public readonly string ApiKey;
```

Comment:
Why you use public readably field instead of public readonly property? A little bit non-conventional way...

Reply:
I'm not going to serialize it in any way, and it's just a data-record, so why not? Don't need properties at the moment and it could be introduced asap without changing any calling code